plugins/nf-azure/src/main/nextflow/cloud/azure/config/AzPoolOpts.groovy

```diff
@@ -37,11 +37,11 @@ import nextflow.util.Duration
 class AzPoolOpts implements CacheFunnel {
 
     static public final String DEFAULT_PUBLISHER = "microsoft-azure-batch"
-    static public final String DEFAULT_OFFER = "centos-container"
-    static public final String DEFAULT_SKU = "batch.node.centos 8"
+    static public final String DEFAULT_OFFER = "ubuntu-server-container"
+    static public final String DEFAULT_SKU = "batch.node.ubuntu 20.04"
     static public final String DEFAULT_VM_TYPE = "Standard_D4_v3"
     static public final OSType DEFAULT_OS_TYPE = OSType.LINUX
-    static public final String DEFAULT_SHARE_ROOT_PATH = "/mnt/resource/batch/tasks/fsmounts"
+    static public final String DEFAULT_SHARE_ROOT_PATH = "/mnt/batch/tasks/fsmounts"
     static public final Duration DEFAULT_SCALE_INTERVAL = Duration.of('5 min')
```

Comment:
what's the rationale for this?

Reply:
The mount path is different for `Ubuntu` and `CentOS` based machines, a previous discussion happened in https://github.com/nextflow-io/nextflow/pull/2285#issuecomment-931867098 regarding this topic.


Also from our docs,

```
azure.batch.pools.<name>.fileShareRootPath          |           If mounting File Shares, this is the internal root mounting point. Must be /mnt/resource/batch/tasks/fsmounts for CentOS nodes or /mnt/batch/tasks/fsmounts for Ubuntu nodes (default is for CentOS, requires nf-azure@0.11.0).

```